src/common/telemetry.ts

```diff
@@ -13,12 +13,21 @@ const HEAP_APPID_PROD = '4039408868'; // todo: Dev server, change it back to pro
 const HEAP_ENDPOINT = 'https://heapanalytics.com/api/track';
 const HEAP_HEADERS = { 'Content-Type': 'application/json' };
 
-export async function sendHeapEvent(eventName: string, properties?: any, identity?: string): Promise<void> {
+export async function sendHeapEvent(commandName: string, properties?: any, identity?: string): Promise<void> {
```

Comment:
@noklam but we are not passing commandName at all ?

Reply:
fixed